app/src/main/res/values/styles.xml

```diff
@@ -34,4 +34,4 @@
         <item name="android:textColor">#000</item>
         <item name="android:textSize">19sp</item>
     </style>
-</resources>
+</resources>
```

Comment:
Revert

Reply:
Done 👍

Reply:
Not done

Reply:
I changed this but after pressing Alt+Ctrl+L it reformats the code. But this time I revert it.

Reply:
Still not reverted

Reply:
![Screenshot 2019-04-03 at 3 22 10 PM](https://user-images.githubusercontent.com/20669217/55469990-58caae00-5624-11e9-8182-d90ef8aef2e2.png)

Reply:
This file should not show in File Changes, that's it